docs/src/content/extra/detailed.md

```diff
@@ -0,0 +1,206 @@
+---
+title: 'Eludris Detailed'
+description: 'A detailed rundown and initial spec of how Eludris is to eventually function'
+order: 0
+---
+
+## Overview
+
+The main goal with Eludris to provide a uniquely fresh but not entirely new experience. One where anyone can find (or create) a place focused around their interests while having a couple of fundamental policies that greatly improve users' experience.
```

Comment:
I think you forgot an "is" -- "The main goal with Eludris **is** to provide <...>".